.github/workflows/consumers.yaml

```diff
@@ -0,0 +1,32 @@
+# Build the latest versions of applications that consume Dyninst
+
+name: Build Consumers
+
+on:
+  schedule:
+    - cron: '0 3 * * 1' # Monday at 3AM
+
+jobs:
+  build:
+    strategy:
+      fail-fast: false
+      matrix:
+        consumer: [
+          "hpctoolkit"
+          # must+stackwalker  - python is broken
+          # stat              - takes >6hrs
+          # extrae+dyninst    - not yet tested, may not support dyninst >10.0.0
+          # omnitrace         - Needs updated cmake
```

Comment:
What does `Needs updated cmake` mean? It may be missing some of the latest versions but that is fine for your use case I think. Or you could potentially target `omnitrace@main`.

Reply:
It was my understanding that omnitrace and timemory wwere using a custom fork of Dyninst, so testing them here wouldn't necessary reflect changes made in this repo.

Reply:
Omnitrace has a custom submodule for when it is built for packaging but it fully supports using an external Dyninst installation and the spack recipe for omnitrace is written such that it always relies on the Dyninst installed via spack. 

Don't worry about timemory.